src/components/Ticket/Ticket.tsx

```diff
@@ -1,9 +1,5 @@
 import { CSSProperties, useContext } from "react";
```

Comment:
@kei95 Do you have any preference for the order of imports?

some people like React first and in alphabetical order and some people also like the break line between React and other public libraries and another break line before own library.

Reply:
Good point, definitely React first. I'd usually structure it like the following! Lemme know if it doesn't really algin to your preference since I'm curious😄 

```
import React from 'react';
import Dependencies from 'some-dependency';
import Dependencies from 'some-dependency';
import Dependencies from 'some-dependency';

import myOwnComponent from '../path/to/component';
```